tests/expectations/exceptionWithInvalidCode.json

```diff
@@ -1,9 +1 @@
-{
-  "errors": [
-    {
-      "status": "Some error occurred",
-      "code": "600",
-      "title": "An exception occurred"
-    }
-  ]
-}
+{"errors":[{"status":"Some error occurred","code":"600","title":"An exception occurred"}]}
```

Comment:
why is this structured like this now?! 🤔was way better to read before, wasn't it?!

Reply:
We structured the file like this because of the string comparison in the tests, but as discussed Ardit will take care of this in Issue #20 (he will make sure that we don't do string comparisons anymore and rather only compare the important information).